backend/push.js

```diff
@@ -119,7 +119,7 @@ export const subscriptionInfoWrapper = (subcriptionId: string, subscriptionInfo:
         let salt
         let uaPublic
 
-        return function (this: Object) {
+        return function (/* this: Object */) {
```

Comment:
Why was this commented out? 🤔 

(I think @corrideat added this for some specific reason)

Reply:
It caused a syntax error, complaining about `function (this)` being invalid code, like if only the `: Object` part had been removed when transpiling JS with Flow annotations to plain JS.

Reply:
@snowteamer What complained about a syntax error? It's definitely a JavaScript syntax error (`this` isn't allowed as an identifier), but it's not a Flow (or TypeScript syntax error), and this annotation is used to represent the type of `this`.

For example:

```
type Foo = {
  bar: number
}

function usesFoo(this: Foo) {
  // Ok, because we've declared `this` should be of type `Foo`
  alert(this.bar)
}

function usesFoo() {
  // Not ok, because this is assumed to be `window` / `self` / `globalThis`,
  // and `.bar` is an invalid property of the global object.
  alert(this.bar)
}

```

Reply:
Replying to myself, if the error is Flow-related, it could be because packages such as `@babel/plugin-syntax-flow` are being downgraded with this PR, and the older version maybe didn't have support for these `this` annotations.